### PR TITLE
Add python yarn package script

### PR DESCRIPTION
**Pull request by jenniew**
Add script to create virtual environment.

Related issue: https://github.com/intel-analytics/zoo/issues/215

### CODE REVIEW

**Comment by zhichao-li** [comment]
The packing script looks great. 
To make it  more straight forward to the user, can we add some description about how to use the `python_package.sh` together with `spark-submit-with-zoo.sh` on `scripts/README.md` ?

**Comment by jenniew** [comment]
updated

File: scripts/README.md [hunk 1]
```diff
@@ -41,3 +41,55 @@ After setting the necessary environment variables above, you can run those scrip
     path_to_your_python_script_of_model
 ```
 Note that not all the parameters are required. You only need to set the necessary ones.
+
+    * For YARN cluster:
```
Comment (hkvision) [inline scripts/README.md]: Something wrong with the format rendered by Github. Have a fix?

Comment (jenniew) [reply scripts/README.md]: fixed

**Pull request merged**